NewHorizons/Patches/ToolPatches/ToolModeSwapperPatches.cs

```diff
@@ -21,7 +23,9 @@ public static bool ToolModeSwapper_EquipToolMode(ToolModeSwapper __instance, Too
                 mode == ToolMode.Probe ||
                 mode == ToolMode.SignalScope ||
                 mode == ToolMode.Translator;
-            var isInShip = UnityEngine.Object.FindObjectOfType<ShipCockpitController>()?._playerAtFlightConsole ?? false;
```

Comment:
xen says can use PlayerState.AtFlightConsole here. i cant make edits directly to dev branch and dont want to pr for one line